frontend/src/metabase/embedding-sdk/components/SimpleDataPicker/SimpleDataPicker.stories.tsx

```diff
@@ -61,17 +61,17 @@ const SUPER_LONG_OPTIONS = createOptions([
 function createOptions(optionNames: string[]): Option[] {
   return optionNames.map((name, index) => ({
     id: index + 1,
-    display_name: name,
+    name: name,
```

Comment:
This is from testing the actual search endpoint and there's no `display_name` property.